src/Tribe/Log/Service_Provider.php

```diff
@@ -22,6 +22,8 @@ class Service_Provider extends \tad_DI52_ServiceProvider {
 	 * @since 4.9.16
 	 */
 	public function register() {
+		$this->container->singleton( 'log', $this );
+		$this->container->singleton( static::class, $this );
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: c5bce384583a3ba28d7fc42021a6d36a</sub>